deltametrics/plot.py

```diff
@@ -1118,6 +1119,9 @@ def aerial_view(elevation_data, datum=0, ax=None, ticks=False,
     colorbar_kw
         Dictionary of keyword args passed to :func:`append_colorbar`.
 
+    return_im : bool, optional
```

Comment:
not disagreeing with the choice of `return_im` here, but we will want to be consistent through the package. I think `im` makes sense as a general name for the image plotted by things, just wanted to make sure you thought about it in this context. Only alternative that comes to mind would be `return_image`, but `im` may actually be better because it makes it clearer `im` is an object (not just an array or something) from `imshow`.

Reply:
I think the only other place we do this is for the widths/depths of channels which use `return_widths` and `return_depths` respectively. So I agree, here either `return_im` or `return_image` could work

Reply:
we have some [optional returns in other unrelated functions too](https://github.com/DeltaRCM/DeltaMetrics/blob/b82c3e46b2a2730aab44f9904734108ca77a549e/deltametrics/plan.py#L687) and [here too](https://github.com/DeltaRCM/DeltaMetrics/blob/b82c3e46b2a2730aab44f9904734108ca77a549e/deltametrics/plan.py#L882). These are more specific to what is returned though. 

I think it's fine as is, unless you have any unifying idea...